robot/packages/utils.go

```diff
@@ -22,6 +22,10 @@ import (
 	rutils "go.viam.com/rdk/utils"
 )
 
+const (
+	windowsOS = "windows" // constant for Windows OS identifier
```

Comment:
Could put the whole `if runtime.GOOS == "windows"` in a helper, too. I imagine it's helpful to know where all the windows checks are (I think I saw a spreadsheet of it recently?)

Reply:
yep, i will do it in a follow up PR.